Sources/Client/WebSocket/Provider/StarscreamWebSocketProvider.swift

```diff
@@ -35,7 +35,7 @@ final class StarscreamWebSocketProvider: WebSocketProvider {
     }
     
     func disconnect() {
-        webSocket.disconnect(forceTimeout: 0)
+        webSocket.disconnect()
```

Comment:
The function we use here is `disconnect(closeCode: UInt16 = CloseCode.normal.rawValue)` but as you see it takes a `closeCode`. Do we need to update `WebSocketProvider.disconnect()` to require this value as an input? (maybe wrapped into some enum)

Reply:
I think it's fine as is.